ocaml/xapi/xapi.ml

```diff
@@ -444,7 +447,60 @@ let attempt_pool_hello my_ip =
               [localhost_uuid] ;
             Some Permanent
         | `ok ->
-            None
+            let obj_uuid = Helpers.get_localhost_uuid () in
+            let message_name, _ =
+              Api_messages
+              .xapi_startup_blocked_as_version_higher_than_coordinator
+            in
+            let existing_alerts =
+              Client.Client.Message.get_all_records ~rpc ~session_id
```

Comment:
This is potentially a lot of data that gets transferred from the coordinator, every time xapi starts up.

Reply:
To be honest, I would just send the alert if needed, and not worry about previous alerts. This is going to be an unusual situation anyway.

Reply:
I looked at the context here again, how about doing `Xapi_host.set_emergency_mode_error` when `Xapi_globs.slave_emergency_mode` is `true`?
It seems in this circumstance, this is an existing way to expose the error.

Reply:
The alert would be sent every 1 minutes, I think we had to avoid sending so many duplicated alerts when the xapi in the pool member is updated and restarted earlier than coordinator. While there seems not an easy way to achieve this.
Had a discussion with @minglumlu , I think Ming's suggestion is great, set emergency mode the same as other "hello error" seems a good choice.
But currently, XC has no way to check the host's emergency mode, as the only api: "local_assert_healthy" is not exported and _R_LOCAL_ROOT_ONLY.

Reply:
Changed to alert only once after xapi restart, and will not remove the alert(s) after it startup successfully.